src/platforms/alexa/apis/ApiBase.ts

```diff
@@ -0,0 +1,49 @@
+import { RequestEnvelope } from "ask-sdk-model";
+import * as debug from "debug";
+import * as _ from "lodash";
+import * as rp from "request-promise";
+
+const alexalog: debug.IDebugger = debug("voxa:alexa");
+
+export class ApiBase {
+  public errorCodeSafeToIgnore: number = 0; // the code error to ignore on checkError function
+  public tag: string = ""; // the class reference for error logging
+  public voxaEvent: any; // the event as sent by the service
```

Comment:
instead of using a `voxaEvent` perhaps it's better to use the `rawEvent` which has a type of RequestEnvelope if i'm not mistaken. In any case, let's try and not use `any` when possible